scripts/utils/actions/chainlink.js

```diff
@@ -0,0 +1,37 @@
+const BN = require('bignumber.js')
+
```

Comment:
Why the file test-integration/utils/actions/blockchain.js was moved? the same for atmGovernance.js. We have all actions in the test-integration/utils/actions folder.

Reply:
I moved all the actions to the `scripts` folder because they are being used by both the scripts and the integration tests. It makes more sense to have them in the scripts folder since they are a script action that can be used by tests.